### PR TITLE
Updated Edgelet Dev Document

### PR DESCRIPTION
**Pull request by nimanch**
Updated Edgelet Dev Document on Updated Instructions to Run Edge Daemon Locally

### CODE REVIEW

File: edgelet/doc/devguide.md [hunk 1]
```diff
@@ -168,11 +168,51 @@ To run `iotedged` locally:
...
+
+    - Windows
+      ```powershell
+      Copy-Item .\edgelet\contrib\config\windows\config.yaml -Destination $env:IOTEDGE_HOMEDIR
+      ```
+
+3. Update the Device Connection String in `config.yaml` to take the device connection string from your iot hub. To Create a device in IoT Hub - Please see [Quickstart](https://docs.microsoft.com/en-us/azure/iot-edge/quickstart-linux?view=iotedge-2020-11)
+
+4. In Linux, Make sure $USER is added to docker group ( Make sure Moby-Engine is installed in your machine by following instructions [here](https://docs.microsoft.com/en-us/azure/iot-edge/how-to-install-iot-edge?view=iotedge-2020-11#install-a-container-engine))
+
```
[the remark this answers is not in the document]
Comment (nimanch) [reply edgelet/doc/devguide.md:191]: Moved

File: edgelet/doc/devguide.md [hunk 2]
```diff
@@ -168,11 +168,51 @@ To run `iotedged` locally:
...
+        Copy-Item Resolve-Path
         ```
 
-1. Create a `config.yaml`. It's okay to create this under the `IOTEDGE_HOMEDIR` directory.
+2. Copy the configy.yaml file to the `IOTEDGE_HOMEDIR` directory.
 
-1. Run the daemon with the `IOTEDGE_HOMEDIR` environment variable set and with the path to the `config.yaml`
+    - Linux / MacOS
+      ```sh
+      sudo cp edgelet/contrib/linux/config.yaml $IOTEDGE_HOMEDIR
```
[the remark this answers is not in the document]
Comment (nimanch) [reply edgelet/doc/devguide.md]: Fixed

File: edgelet/doc/devguide.md [hunk 3]
```diff
@@ -168,11 +168,51 @@ To run `iotedged` locally:
...
+      ```
+
+    - Windows
+      ```powershell
+      Copy-Item .\edgelet\contrib\config\windows\config.yaml -Destination $env:IOTEDGE_HOMEDIR
+      ```
+
+3. Update the Device Connection String in `config.yaml` to take the device connection string from your iot hub. To Create a device in IoT Hub - Please see [Quickstart](https://docs.microsoft.com/en-us/azure/iot-edge/quickstart-linux?view=iotedge-2020-11)
+
+4. In Linux, Make sure $USER is added to docker group ( Make sure Moby-Engine is installed in your machine by following instructions [here](https://docs.microsoft.com/en-us/azure/iot-edge/how-to-install-iot-edge?view=iotedge-2020-11#install-a-container-engine))
```
[the remark this answers is not in the document]
Comment (nimanch) [reply edgelet/doc/devguide.md]: Yes, I will keep that as part of separate task and PR

File: edgelet/doc/devguide.md [hunk 4]
```diff
@@ -168,11 +168,51 @@ To run `iotedged` locally:
...
+    grep docker /etc/group
+    ```
+
+    Add User to Docker Group
+
+    ```sh
+    sudo usermod -aG docker $USER
+    ```
+
+    Restart docker
```
[the remark this answers is not in the document]
Comment (nimanch) [reply edgelet/doc/devguide.md]: I verified that docker restart is not actually needed. Removed the comment

**Review by scb01 — APPROVED** [review]

**Pull request merged**